src/geant4/app/include/ExN01DetectorConstruction.hh

```diff
@@ -33,7 +34,7 @@ class ExN01DetectorConstruction : public G4VUserDetectorConstruction {
   // particle filters for tallies
   std::map<std::string, G4SDParticleFilter*> particle_filters;
 
-  void BuildParticleFilter(std::string particle_name);
+  void BuildParticleFilter(std::vector<std::string> particle_name);
```

Comment:
These don't seem to have anything to do with the changes to the PyNE `MaterialLibrary` API. Is it part of the Geant4 API update?

Reply:
no it is part of the PyNE::Tally API :)

Reply:
Ah, I see. :+1: